packages/commercetools/theme/components/Checkout/VsfShippingProvider.vue

```diff
@@ -88,13 +87,12 @@ export default {
     const isShippingMethodStepCompleted = ref(false);
     const loading = ref(false);
     const shippingMethods = ref([]);
-    const selectedShippingMethod = ref({});
     const { $ct } = useVSFContext();
-    const { cart, setCart } = useCart();
+    const { cart } = useCart();
+    const { response: selectedShippingMethod, save, error: errorShippingProvider, loading: loadingShippingProvider } = useShippingProvider();
```

Comment:
do we need the renames here?

Reply:
For `error` and `loading` yes, because we have refs called like that inside component. About `response`, it is just for readability